conda-forge.yml

```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
Currently `linux_ppc64le` is not supported on this feedstock. This would depend on building `libcufile` on `linux_ppc64le` ( https://github.com/conda-forge/libcufile-feedstock/pull/10 )

Let's drop `linux_ppc64le` from `build_platform`. This can be revisited if `linux_ppc64le` builds become available

```suggestion
```